tests/setup.py

```diff
@@ -15,7 +15,7 @@
     install_requires=[
         'ducktape@git+https://github.com/redpanda-data/ducktape.git@7ba83f32d5f265aad955a31096dcc0c97d96c0ce',
         'prometheus-client==0.9.0', 'pyyaml==5.3.1', 'kafka-python==2.0.2',
-        'crc32c==2.2', 'confluent-kafka==1.7.0', 'zstandard==0.15.2',
+        'crc32c==2.2', 'confluent-kafka==2.0.2', 'zstandard==0.15.2',
```

Comment:
@BenPope (as the author of https://github.com/redpanda-data/redpanda/pull/7431#issuecomment-1513703044) is this version update ok for 22.2.x?

Reply:
explanation/motivation is in the last commit https://github.com/redpanda-data/redpanda/pull/11061/commits/5b883597f05b183f8a2afcef098341b3f6786e59

Reply:
> @BenPope (as the author of [#7431 (comment)](https://github.com/redpanda-data/redpanda/pull/7431#issuecomment-1513703044)) is this version update ok for 22.2.x?

I think so, I seem to recall this is the change that's necessary for that version: https://github.com/redpanda-data/redpanda/commit/fdcedb79249a5a694b52a78302d44c45bf4e5768